static/style.css

```diff
@@ -1,5 +1,381 @@
 
+:root {
+    --main-color: #ecf0f1;
+    --point-color: #555;
+    --size: 5px;
+  }
+
+  .loader {
+    background-color: var(--main-color);
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    top: 0; left: 0;
+    display: flex;
+    align-items: center;
+    align-content: center;
+    justify-content: center;
+    z-index: 100000;
+  }
+
+  .loader__element {
+    border-radius: 100%;
+    border: var(--size) solid var(--point-color);
+    margin: calc(var(--size)*2);
+  }
+
+  .loader__element:nth-child(1) {
+    animation: preloader .6s ease-in-out alternate infinite;
+  }
+  .loader__element:nth-child(2) {
+    animation: preloader .6s ease-in-out alternate .2s infinite;
+  }
+
+  .loader__element:nth-child(3) {
+    animation: preloader .6s ease-in-out alternate .4s infinite;
+  }
+
+  @keyframes preloader {
+    100% { transform: scale(2); }
+  }
+
+
+
 body {
     background-color: rgba(253, 182, 88, 0.763); /* Cream-colored background */
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
     min-height: 100vh; /* Minimalna wysokość strony na 100% */
 }
+
+/* Wrapper – obejmuje całą stronę */
+.wrapper {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh; /* Minimum 100% wysokości ekranu */
```

Comment:
Use `%` instead of `vh` if possible. To make `height: 100%` work, you might need to add it to `body` and `html`.

```
html, body  {
  height: 100%;
}
```